pkg/controller/volume/cache/actual_state_of_world_test.go

```diff
@@ -136,15 +137,15 @@ func Test_AddVolumeNode_Positive_ExistingVolumeExistingNode(t *testing.T) {
 		t.Fatalf("len(attachedVolumes) Expected: <1> Actual: <%v>", len(attachedVolumes))
 	}
 
-	verifyAttachedVolume(t, attachedVolumes, generatedVolumeName1, volumeName, nodeName, false /* expectedSafeToDetach */, false /* expectNonZeroDetachRequestedTime */)
+	verifyAttachedVolume(t, attachedVolumes, generatedVolumeName1, string(volumeName), nodeName, true /* expectedMountedByNode */, false /* expectNonZeroDetachRequestedTime */)
```

Comment:
Because the value happens to be the same as `UniqueDeviceName` for these tests, but may be different. `attachedVolume.VolumeSpec.Name()` could be any arbitrary string--it's actual type is a string not `UniqueDeviceName`